addons/website/static/src/js/content/redirect.js

```diff
@@ -16,70 +14,15 @@ document.addEventListener('DOMContentLoaded', () => {
     }
 
     if (!window.frameElement) {
-        const websiteId = document.documentElement.dataset.websiteId;
-        const {pathname, search} = window.location;
-        const params = new URLSearchParams(search);
-        const enableEditor = params.get('enable_editor');
-        const editTranslations = params.get('edit_translations');
-        params.delete('enable_editor');
-        let newSearch = params.toString();
-        if (newSearch) {
-            newSearch = '?' + newSearch;
-        }
-        const backendPath = `/web#action=website.website_preview&path=${encodeURIComponent(`${pathname}${newSearch}`)}&website_id=${websiteId}`;
-
-        const autoredirectToBackendAction = enableEditor || editTranslations;
-        if (autoredirectToBackendAction) {
+        // Auto redirect to frontend if edit/translation mode is requested
+        const currentUrl = new URL(window.location.href);
+        currentUrl.pathname = `/@${currentUrl.pathname}`;
+        if (currentUrl.searchParams.get('enable_editor') || currentUrl.searchParams.get('edit_translations')) {
```

Comment:
```currentUrl.pathname = `/@${currentUrl.pathname}`;``` can be inside the `if` :wink: Actually I thought that computing the `/@/` URL in the XML template would allow to get rid of it by using the link element here... but I guess you don't need to search for it in JS anymore so that would not be better in the end :grimacing: So just this line inside the `if` and it would be nice IMO :+1: I know: nitpicking :grimacing:

Reply:
Sometimes there is nitpicking, but in this case it is not, it has nothing to do outside the if so good to mention it